org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/OrganizeImportsHandler.java

```diff
@@ -173,6 +176,21 @@ public static TextEdit wrapStaticImports(TextEdit edit, CompilationUnit root, IC
 		return edit;
 	}
 
+	private static boolean hasImport(ICompilationUnit unit, SimpleName node) throws JavaModelException {
+		IImportDeclaration[] iImportDeclarations = unit.getImports();
+		for (IImportDeclaration importDeclaration : iImportDeclarations) {
+			String elementName = importDeclaration.getElementName();
+			int lastDot = elementName.lastIndexOf('.');

```

Comment:
I'd probably use Signature.getSimpleName(elementName) assuming it does exactly what you need.